tests/test_modeling_common.py

```diff
@@ -3495,8 +3495,16 @@ def test_sdpa_can_dispatch_composite_models(self):
 
                 vision_model_sdpa = getattr(model, vision_model_name)
                 language_model_sdpa = getattr(model, language_model_name)
-                text_attn = "sdpa" if language_model_sdpa._supports_sdpa else "eager"
-                vision_attn = "sdpa" if vision_model_sdpa._supports_sdpa else "eager"
+                text_attn = (
+                    "sdpa"
+                    if (language_model_sdpa._supports_sdpa or language_model_sdpa._supports_attention_backend)
+                    else "eager"
+                )
+                vision_attn = (
+                    "sdpa"
+                    if (vision_model_sdpa._supports_sdpa or vision_model_sdpa._supports_attention_backend)
+                    else "eager"
+                )
```

Comment:
ah.. I see you updated here as well

Can we have a case when FA2 is not supported while SDPA is supported 🤔  I guess it might be the case for Flex - SDPA at least